From b28c9ec04acf9a63d826151cba38178a0a0175ef Mon Sep 17 00:00:00 2001
From: WANG Xuerui <git@xen0n.name>
Date: Sun, 8 Aug 2021 00:13:23 +0800
Subject: [PATCH 26/27] LoongArch: Bump minimum kernel version to 5.14.0

There is no LoongArch support in any earlier upstream kernels, so first
bump to 5.14.0 where the earliest open-source LoongArch appeared on top
of. This version of glibc is unlikely to be paired with early
non-open-source vendor kernels, so the change should be harmless.
---
 sysdeps/unix/sysv/linux/loongarch/configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sysdeps/unix/sysv/linux/loongarch/configure.ac b/sysdeps/unix/sysv/linux/loongarch/configure.ac
index fef4f4d2ae..5709a0ed43 100644
--- a/sysdeps/unix/sysv/linux/loongarch/configure.ac
+++ b/sysdeps/unix/sysv/linux/loongarch/configure.ac
@@ -2,7 +2,7 @@ sinclude(./aclocal.m4)dnl Autoconf lossage
 GLIBC_PROVIDES dnl See aclocal.m4 in the top level source directory.
 # Local configure fragment for sysdeps/unix/sysv/linux/loongarch.
 
-arch_minimum_kernel=4.15.0
+arch_minimum_kernel=4.19.0
 
 libc_cv_loongarch_int_abi=no
 AC_EGREP_CPP(4 4 4, [__SIZEOF_INT__ __SIZEOF_LONG__ __SIZEOF_POINTER__
-- 
2.30.1

